buck.iml

```diff
@@ -92,5 +92,8 @@
     <orderEntry type="library" name="jna-4.2.0" level="project" />
     <orderEntry type="library" name="jna-platform-4.2.0" level="project" />
     <orderEntry type="library" name="rocksdbjni-5.1.2" level="project" />
+    <orderEntry type="library" name="kotlin-compiler-1.1" level="project" />
```

Comment:
Considering kotlin versions evolve rapidly, it may not make sense to bundle kotlin jars with buck for every release. Maybe its best if we let consumers point to the kotlin jars they want as `remote_file`s or jars etc.?